src/components/CustomSelectDropdown/CustomSelectDropdown.css

```diff
@@ -1,10 +1,10 @@
 .CustomSelectDropdown__options {
   overflow: hidden;
-  background-color: var(--modal_card_background);
+  background-color: var(--vkui--color_background_content);
```

Comment:
Нужно использовать [промежуточный вариант](https://github.com/VKCOM/VKUI/blob/master/VKUI_TOKENS_MIGRATION_GUIDE.md#%D0%BF%D1%80%D0%BE%D0%BC%D0%B5%D0%B6%D1%83%D1%82%D0%BE%D1%87%D0%BD%D1%8B%D0%B9-%D0%B2%D0%B0%D1%80%D0%B8%D0%B0%D0%BD%D1%82)

```suggestion
  background-color: var(--modal_card_background, var(--vkui--color_background_content);
```
<img width="1050" alt="image" src="https://user-images.githubusercontent.com/14944123/163384284-ccc837b1-d269-4571-b36f-4eeaec4d73aa.png">